main.cpp

```diff
@@ -0,0 +1,17 @@
+#include "character.h"
+#include "game.h"
+
+
+
+
+int main(int argc, char *argv[]) {
+	 Character ch01(argv[1], std::stoi(argv[2]), std::stoi(argv[3]));
+	 Character ch02(argv[4], std::stoi(argv[5]), std::stoi(argv[6]));
+	Game* G= new Game(&ch01,&ch02);		

```

Comment:
Miért dinamikusan van létrehozva? Felesleges, de akkor meg már fel is kéne szabadítani.